org.gramps_project.Gramps.yml

```diff
@@ -19,48 +17,26 @@ finish-args:
 # for gui
   - --socket=wayland
   - --socket=fallback-x11
-  - --share=ipc
```

Comment:
not sure why you'd remove this it is needed for x11 even as fallback

Reply:
Thank you.  I must have misread the flatpak docs because I thought it was only needed for graphics intensive apps that used x11, and the Gramps flatpak worked without it.  But when I checked just now, the docs said it was necessary https://docs.flatpak.org/en/latest/sandbox-permissions.html so I'll add it back in.